common/src/View/InputState.h

```diff
@@ -111,6 +113,14 @@ class InputState
    * Number of "lines" to scroll vertically.
    */
   float scrollY() const;
+  /**
```

Comment:
If we decide to generalize the `WheelAxis` enum, we should also choose clearer names such as `pinchAmount` and `rotateAmount` or something like that.

Reply:
Yeah, that's probably cleaner. I'll rename them.